aspnetcore/getting-started.md

```diff
@@ -24,20 +24,28 @@ uid: getting-started
    On macOS and Linux, open a terminal window. On Windows, open a command prompt.
 
    ```terminal
-   mkdir aspnetcoreapp
-   cd aspnetcoreapp
-   dotnet new web
+   dotnet new razor -o aspnetcoreapp
    ```
     
 4. Run the app.
 
-   The `dotnet run` command builds the app first if needed.
+    Use the following commands to run the app:
```

Comment:
I think having:

3. Change your current directory to *aspnetcoreapp*
    ```terminal
    cd aspnetcoreapp
    ```

4. Run the app
    ```terminal
    dotnet run
    ```

will be better!

Reply:
It was that way before and was requested to change to a shorter format.

Reply:
Sorry, my bad! I didn't see that version :(